### PR TITLE
Added introduction and cleaned up Doc2vec-IDMB notebook

### PR DESCRIPTION
**Pull request by pahdo**
Added introduction, motivation, etc. and cleaned up this tutorial, and then re-ran the notebook.

### CODE REVIEW

File: docs/notebooks/doc2vec-IMDB.ipynb [hunk 1]
```diff
@@ -916,11 +928,18 @@
...
-   "execution_count": null,
-   "metadata": {
-    "collapsed": true
-   },
-   "outputs": [],
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [
+    {
+     "ename": "SyntaxError",
```
Comment (menshikh-iv) [inline docs/notebooks/doc2vec-IMDB.ipynb]: Please check this

**Comment by menshikh-iv** [comment]
Sorry for late answer @pahdo, please look to my comment and fix it, after I'll merge your PR.

**Comment by pahdo** [comment]
Thanks for looking over my PR! I fixed the issue you mentioned.

**Comment by menshikh-iv** [comment]
Thank you @pahdo, congratz with your first contribution :1st_place_medal:

**Pull request merged**